[2024-11-01T18:24:45.950+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-11-01T18:24:46.167+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-11-01T18:24:46.175+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-11-01T18:24:46.741+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_gcs_task> on 2024-03-01 00:00:00+00:00
[2024-11-01T18:24:46.854+0000] {standard_task_runner.py:57} INFO - Started process 395 to run task
[2024-11-01T18:24:46.854+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bluebikes_data_pipeline', 'upload_to_gcs_task', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '811', '--raw', '--subdir', 'DAGS_FOLDER/***_main.py', '--cfg-path', '/tmp/tmp_v2ceve7']
[2024-11-01T18:24:46.912+0000] {standard_task_runner.py:85} INFO - Job 811: Subtask upload_to_gcs_task
[2024-11-01T18:24:48.220+0000] {task_command.py:410} INFO - Running <TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task scheduled__2024-03-01T00:00:00+00:00 [running]> on host f1ce4a924b75
[2024-11-01T18:24:48.942+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Muskan' AIRFLOW_CTX_DAG_ID='bluebikes_data_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-11-01T18:25:14.152+0000] {logging_mixin.py:150} INFO - Error uploading /opt/***/extracted_files/202403-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/blue_bikes_bucket/o?uploadType=resumable&upload_id=AHmUCY1TbXMW7-duwSrNiEhx-GVI-LJzSndgCUdW0CThSALUSVhnAwTtWKmFDtGl73M_mNQB5lhq31B7IZ0t2_y2uVFul85hvvT328bP6oD01GQO (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-01T18:25:27.165+0000] {logging_mixin.py:150} INFO - Error uploading /opt/***/extracted_files/202403-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/blue_bikes_bucket/o?uploadType=resumable&upload_id=AHmUCY32g4Jn6dq898msaFf5WKJ2KxvAjaOg7GatehcVl0i95scvteNe0ns6Ug4qnSmswaYneOg71nkMJXo7Xf1WehLCCbz5B3j18UIffHE7w1qk (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-01T18:25:39.320+0000] {logging_mixin.py:150} INFO - Error uploading /opt/***/extracted_files/202403-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/blue_bikes_bucket/o?uploadType=resumable&upload_id=AHmUCY3eIn6k2wVaLy-fzFnon7ntiCKcFLkgsIR7HnJcemr6UXjFX9K8Vl9uub9m0otgM-HNc21aafi41cmKxYyB1vMzCAcL93yDNX_crMeUulG95A (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-01T18:25:55.896+0000] {logging_mixin.py:150} INFO - Error uploading /opt/***/extracted_files/202403-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/blue_bikes_bucket/o?uploadType=resumable&upload_id=AHmUCY0MdCbwegsE0MMP0sQT-_PKkNiMvhZHwpnz5bKW2uR4J-hIeBzzQs72x5I_FWrbmMdqMiU_W9lnCRyBARQIJLZU3F3dSR1mZwhSJhF7magYbQ (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-01T18:26:19.319+0000] {logging_mixin.py:150} INFO - Uploaded /opt/***/extracted_files/202403-bluebikes-tripdata.csv to gs://blue_bikes_bucket/202403-bluebikes-tripdata.csv
[2024-11-01T18:26:19.352+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-11-01T18:26:19.656+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=bluebikes_data_pipeline, task_id=upload_to_gcs_task, execution_date=20240301T000000, start_date=20241101T182445, end_date=20241101T182619
[2024-11-01T18:26:19.866+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-11-01T18:26:20.008+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
